style.css

```diff
@@ -0,0 +1,150 @@
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  background-image: url("./images/background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  justify-content: center;
+}
+
+body h1 {
+  font-size: 50px;
+}
+
+body a {
+  text-decoration: none;
+  font-weight: bold;
+  color: cornflowerblue;
+}
+
+.header-title {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  padding-right: 0.2em;
+  cursor: pointer;
+}
+
+.stroke {
+  width: 20px;
+  height: 3px;
+  background-color: #6070ff;
+  border-radius: 8px;
+  list-style-type: none;
+}
+
+.mobile-container {
+  max-width: 100%;
+  margin: auto;
+  height: 500px;
+}
+
+.topnav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  padding: 10px;
+}
+
+.topnav a {
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.image-container a {
+  display: flex;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.topnav a.icon {
+  position: relative;
+}
+
+.topnav a.icon i {
+  color: white;
+  font-size: 10px;
+}
+
+#myLinks {
+  display: none;
+  position: absolute;
+  top: 56px;
+  right: 0;
+  background-color: #333;
+  z-index: 1;
+}
+
+#myLinks a {
+  color: white;
+  text-decoration: none;
+  display: block;
+  font-size: 17px;
+}
+
+#myLinks a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.content {
+  padding: 16px;
+}
+
+section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: left;
+  margin-bottom: 1rem;
+}
+
+footer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  color: white;
+}
+
+.image-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] Please I highly recommend sticking to the required [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=HQA60gDOdof1M29t-0). 😄

_**REQUIRED FIGMA DESIGN**_ 

![Screenshot (73)](https://user-images.githubusercontent.com/78104544/235216519-103b0aaf-8dd2-473b-b906-652b58c1f50c.png)

_**YOUR DESIGN**_

- [ ] Please make changes to the `image-container` class which contains the social media icons to fit the required figma design. 😄 

![Screenshot (258)](https://user-images.githubusercontent.com/78104544/235216589-91402e06-9593-480d-a7e2-aec032ff1f86.png)